lang/javascript.py

```diff
@@ -72,30 +72,4 @@ def CursorText(s):
     '(op (power | exponent) | to the power [of])': ' ** ',
     'op and': ' && ',
     'op or': ' || ',
-
-    # commands for express
```

Comment:
I wonder if there is a good way to keep these. I think they would be very nice for someone starting who happens to be using these. Maybe leave them in but commented out? they are pretty unique so even left in, they shouldn't cause too much trouble.

Reply:
I've currently got my own repository going on with all of my web development and language snippets. The file structure is ugly right now, so I need to factor that. I took out all of those snippets and dumped them in a temporary file for now. However, I should get around to working them in soon. You can check it out here if you want: https://github.com/justin-cook-developer/Personal-Talon-Snippets.

Reply:
Sounds good, thanks!

Reply:
Not to mention that they are useless without my VS code snippets. The text that those snippets maps to, triggers an editor snippet.